web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.utils.ts

```diff
@@ -35,25 +41,86 @@ export const getDisplayAdmin = (address?: string): User | undefined => {
   };
 };
 
-export const getDisplayDeveloper = (
-  metadata?: Partial<ProjectMetadataIntersectionLD>,
-  party?: Maybe<PartyFieldsFragment>,
-): User | undefined => {
-  if (!metadata) return;
-  if (metadata?.['regen:projectDeveloper']) {
-    return getDisplayParty('regen:projectDeveloper', metadata, party);
-  }
-  // Possibly temporary: CFC case
-  const projectOperator = (metadata as Partial<CFCProjectMetadataLD>)?.[
-    'regen:projectOperator'
-  ];
-  if (projectOperator) {
-    return {
-      name: projectOperator?.['schema:name'],
-      type: 'ORGANIZATION',
-      link: projectOperator?.['schema:url']?.['@value'] || '',
-    };
-  }
+type AnchoredMetadata = {
+  projectName?: string;
+  area?: string;
+  areaUnit?: string;
+  placeName?: string;
+};
+
+export const getAnchoredMetadata = (
```

Comment:
nit; this name felt misleading to me. when i read "get" i think that we are needing to make request somewhere. to me this functions task seems like more of a "parse". so maybe `parseAnchoredMetadata` is an improvement?